packages/react-dom/src/__tests__/ReactDOMFiber-test.js

```diff
@@ -1332,44 +1242,16 @@ describe('ReactDOMFiber', () => {
       }
     }
 
-    ReactDOM.render(<Parent />, container);
-    inputRef.current.dispatchEvent(
-      new MouseEvent('click', {
-        bubbles: true,
-      }),
-    );
-    expect(didCallOnChange).toBe(true);
-  });
-
-  it('unmounted legacy roots should never clear newer root content from a container', () => {
```

Comment:
Deleted legacy only test, new roots can't be updated after they're unmounted, which we have tests elsewhere for. Still exists in `ReactDOMLegacyFiber`